NuGet.Services.Dashboard.Operations/Infrastructure/SecretConfigurationManager.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Configuration;
+using NuGet.Services.KeyVault;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NuGet
+{
+    public static class SecretConfigurationManager
+    {
+        private static ISecretInjector _secretInjector;
+        private static ISecretReader _keyVaultReader;
+
+        public static void Init()
+        {
+            var vaultName = ConfigurationManager.AppSettings["VaultName"];
+            var clientId = ConfigurationManager.AppSettings["ClientId"];
+            var certificateThumbprint = ConfigurationManager.AppSettings["CertificateThumbprint"];
+            var validateCertificate = ConfigurationManager.AppSettings["ValidateCertificate"].Equals("true", StringComparison.InvariantCultureIgnoreCase) ? true : false;
```

Comment:
>validateCertificate [](start = 16, length = 19)

I would prefer  validateCertificate to always be true (no setting to disable it).
If u feel strongly about it, use bool.Parse